packages/auth/src/components/ReCaptcha.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC, useEffect } from 'react';
+import { loadReCaptcha, ReCaptcha as Captcha } from 'react-recaptcha-v3';
+import { AuthState } from '../Api';
+import { useAuth } from '../hooks';
+
+interface IReCaptchaProps {
+  callback(token: string): void;
+  action: string;
+}
+
+const stateMapper = ({ captchaState }: AuthState) => captchaState;
+
+export const ReCaptcha: FC<IReCaptchaProps> = ({ callback, action }) => {
+  const state = useAuth(stateMapper);
```

Comment:
Use here the `useField` hook and remove changes the value from the components to this component.
In this case, we can use the `useCallback` hook for setting a value to the formik.